Sources/SwiftSyntax/generated/raw/RawSyntaxNodesOP.swift

```diff
@@ -2006,20 +2009,16 @@ public struct RawPrecedenceGroupAttributeListSyntax: RawSyntaxNodeProtocol {
       }
     }
     
-    public init?(_ other: some RawSyntaxNodeProtocol) {
-      if let node = RawPrecedenceGroupRelationSyntax(other) {
+    public init?(_ node: __shared some RawSyntaxNodeProtocol) {
```

Comment:
Nice catch with the missing `__shared` attributes here. 👍🏽

Reply:
Benefits of code deduplication reaped :)